### PR TITLE
DT-23988 Discharge Wizard Update 149 Form 

### PR DESCRIPTION
**Pull request by zacharymorel**
## Issue
https://github.com/department-of-veterans-affairs/va.gov-team/issues/23988

## Description
Adjusted URL to use the older url to the form until VA DBA's can fix the pdf signature on the new form.

## Testing done
Ran e2e, ran unit test, spun up repo locally.

## Screenshots
Screen Shot 2021-05-14 at 12 22 21 PM

## Acceptance criteria
- [x] Download link uses older form.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by zacharymorel** [comment]
@department-of-veterans-affairs/vsa-public-websites-frontend can I get a quick review on this when someone gets a chance?

File: src/applications/discharge-wizard/containers/RequestDD214.jsx [hunk 1]
```diff
@@ -41,7 +41,8 @@ const RequestDD214 = ({ router }) => {
             <div>
               <h4>
                 Download and fill out{' '}
-                <a href="http://www.esd.whs.mil/Portals/54/Documents/DD/forms/dd/dd0149.pdf">
+                {/* <a href="http://www.esd.whs.mil/Portals/54/Documents/DD/forms/dd/dd0149.pdf"> */}
```
Comment (kelsonic) [inline src/applications/discharge-wizard/containers/RequestDD214.jsx:44]: Any reason we wouldn't want to just delete this comment? 🙂

**Review by kelsonic — APPROVED** [review]

File: src/applications/discharge-wizard/containers/RequestDD214.jsx (hunk 1 above)
Comment (zacharymorel) [reply src/applications/discharge-wizard/containers/RequestDD214.jsx:44]: AHHH nice catch

**Comment by kelsonic** [comment]
Sorry last thing, since it's a PDF would you want to put a `download` attribute on that a tag? If you do that, might also want to add an `alt` property like we do [here](https://github.com/department-of-veterans-affairs/content-build/blob/27d441219ac3b4e3d41d642d1bed7b0c6a80c942/src/site/layouts/campaign_landing_page.drupal.liquid) for a11y 🙂

**Comment by zacharymorel** [comment]
> Sorry last thing, since it's a PDF would you want to put a `download` attribute on that a tag? If you do that, might also want to add an `alt` property like we do [here](https://github.com/department-of-veterans-affairs/content-build/blob/27d441219ac3b4e3d41d642d1bed7b0c6a80c942/src/site/layouts/campaign_landing_page.drupal.liquid#L303) for a11y 🙂

Hey, I've figured out that PDF's won't download unless it's the same domain due to CORS.
And example is if you go to => https://www.va.gov/find-forms/?q=health
1. Click on the first result. It should download (It has a domain of va.gov)
2. Now change the sort to Descending.
3. Click the first result, it opens the PDF in a new tab (it has a different domain).

**Comment by kelsonic** [comment]
@zacharymorel true! 💯 Ignore my comment 😂

**Pull request merged**